rules/linux/auditd/lnx_system_shutdown_reboot.yml

```diff
@@ -0,0 +1,36 @@
+title: System shutdown/reboot

```

Comment:
Hello @fitsigor! Thank you for your contribution (: 
You probably should put quotes in the beginning and at the end of the `title` to avoid parsing error.